MolecularNodes/md.py

```diff
@@ -8,25 +8,27 @@
 import bpy
 import numpy as np
 import warnings
+import MDAnalysis as mda
 
 from . import data
 from . import coll
 from . import obj
 from . import nodes
+from .mda import MDAnalysisSession
 
 bpy.types.Scene.mol_import_md_topology = bpy.props.StringProperty(
     name = 'path_topology', 
     description = 'File path for the toplogy file for the trajectory', 
     options = {'TEXTEDIT_UPDATE'}, 
-    default = '', 
+    default = '/nethome/yzhuang/jupyter_playground/blender/md.tpr', 
```

Comment:
Fine for these defaults to remain while you are developing, we'll change them later when we go to potentially merge.